addons/sale_project/tests/test_project_profitability.py

```diff
@@ -174,27 +238,34 @@ def test_project_profitability(self):
             {
                 'revenues': {
                     'data': [
+                        {
+                            'id': 'other_revenues',
+                            'sequence': sequence_per_invoice_type['other_revenues'],
+                            'invoiced': 100.0,
+                            'to_invoice': 0.0,
+                            'action': None
+                        },
                         {
                             'id': invoice_type,
                             'sequence': sequence_per_invoice_type[invoice_type],
                             'to_invoice': sum(service_sols.mapped('untaxed_amount_to_invoice')),
                             'invoiced': sum(service_sols.mapped('untaxed_amount_invoiced')),
                         },
                         {
-                            'id': 'other_revenues',
-                            'sequence': sequence_per_invoice_type['other_revenues'],
+                            'id': 'materials',
+                            'sequence': sequence_per_invoice_type['materials'],
                             'to_invoice': material_order_line.untaxed_amount_to_invoice,
                             'invoiced': material_order_line.untaxed_amount_invoiced,
                         },
                     ],
                     'total': {
                         'to_invoice': sum(service_sols.mapped('untaxed_amount_to_invoice')) + material_order_line.untaxed_amount_to_invoice,
-                        'invoiced': sum(service_sols.mapped('untaxed_amount_invoiced')) + material_order_line.untaxed_amount_invoiced,
+                        'invoiced': sum(service_sols.mapped('untaxed_amount_invoiced')) + material_order_line.untaxed_amount_invoiced+100,
```

Comment:
You can let the `+` breathing a little bit. 🙂 

```suggestion
                        'invoiced': sum(service_sols.mapped('untaxed_amount_invoiced')) + material_order_line.untaxed_amount_invoiced + 100,
```